core/metadata-3.3.md

```diff
@@ -0,0 +1,56 @@
+---
+title: IRCv3.3 Metadata
+layout: spec
+copyrights:
+  -
+    name: "Kiyoshi Aman"
+    period: "2016"
+    email: "kiyoshi.aman@gmail.com"
+---
+## Introduction
+
+This specification incorporates the IRCv3.2 metadata specification and
+incorporates it by reference.
+
+## METADATA
+
+### METADATA PRIORITY
+
+This subcommand provides a mechanism for clients to inform the server what
+priorities to assign to particular keys. The default priority is 'on
+subscription' for all keys. The format of `METADATA PRIORITY` MUST be as
+follows:
+
+    METADATA <Scope> PRIORITY <Priority> :<Keys>
+
+`<Scope>` is one of `global`, `channel`, or `user`, reflecting all subscriptions,
```

Comment:
It is misleading because the word global is used to refer to all people on the network. Consider these examples:

- In reply to LUSERS "Current global users: X Max: Y" means number of all users on the entire network.
- A "global notice" means a notice sent to all users on the server.
- The "Global" service is a service designed to send global notices.
- Nicknames and channel names are globally unique (i.e. on the entire network).

Also in other areas "global" refers to concerning all parts of something, e.g.:
- A global variable in a program is a variable that is visible throughout the entire program.

But Scope being "global" does not subscribe to all users on the network, it only subscribes to other users the client sending the command can see.